CHANGELOG.md

```diff
@@ -1,5 +1,219 @@
+  - key_file: /etc/letsencrypt/live/*.teleport.example.com/privkey.pem
+  - cert_file: /etc/letsencrypt/live/*.teleport.example.com/fullchain.pem
+# ...
+app_service:
+   # Teleport Application Access is enabled.
+   enabled: yes
+   # We've a sample debugger app that'll check
+   # that Teleport Application Access is working
+   # and will output JWT tokens.
+   # https://dumper.teleport.example.com:3080/
+   debug_app: true
+   apps:
+   # Application Access can be used to proxy any HTTP Endpoint
+   # Note: Name can't include any spaces and should be DNS-compatible A-Za-z0-9-._
+   - name: "internal-dashboard"
+     uri: "http://10.0.1.27:8000"
+     # By default Teleport will make this application
+     # available on a sub-domain of your Teleport proxy's hostname
+     # internal-dashboard.teleport.example.com
+     # - thus the importance of setting up wilcard DNS.
+     # If you want, it's possible to setup a custom public url.
+     # DNS records should point to the proxy server.
+     # internal-dashboard.teleport.example.com
+     # Example Public URL for the internal-dashboard app.
+     # public_addr: "internal-dashboard.acme.com"
+     # Optional labels
+     # Labels can be combined with RBAC rules to provide access.
+     labels:
+       customer: "acme",
+       env: "production"
+     # Optional dynamic labels
+     commands:
+     - name: "os"
+       command: ["/usr/bin/uname"]
+       period: "5s"
+     # A proxy can support multiple applications. Application Access
+     # can also be deployed with a Teleport node.
+     - name: "arris"
+       uri: "http://localhost:3001"
+       public_addr: "arris.example.com"
+```
+
+##### Teleport Kubernetes Access
+
+Teleport 5.0 introduces two highly requested features:
+- The ability to connect multiple Kubernetes Clusters to one Teleport Access Plane, greatly reducing operational complexity.
+- We now provide near complete Kubernetes audit log capture [#4526](https://github.com/gravitational/teleport/pull/4526), going beyond our current `kubectl exec`.
+  - For a full overview please review the [Kubernetes RFD](https://github.com/gravitational/teleport/blob/master/rfd/0005-kubernetes-service.md)
+
+To support these changes, we've introduced a new service. This moves Teleport Kubernetes
+support from the `proxy_service` into its own dedicated `kubernetes_service`.
+
+When adding the new Kubernetes service, a new type of join token is required.
+
+```bash
+tctl tokens add --type=kubernetes
+```
+
+Example configuration for the new `kubernetes_service`:
+
+```yaml
+# ...
+kubernetes_service:
+   enabled: yes
+   public_addr: [k8s.example.com:3026]
+   listen_addr: 0.0.0.0:3026
+   kubeconfig_file: /secrets/kubeconfig
+```
+
+#### New "tsh kube" commands
+
+`tsh kube` commands are used to query registered clusters and switch
+`kubeconfig` context:
+
+```sh
+$ tsh login --proxy=proxy.example.com --user=awly
+
+# list all registered clusters
+$ tsh kube clusters
```

Comment:
@russjones @awly just caught this. Please add alias `tsh kube ls` to be consistent with `tsh db ls` and `tsh ls`

Reply:
I've moved this comment to an issue for better tracking https://github.com/gravitational/teleport/issues/4842